app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/ServiceFactory.java

```diff
@@ -9,15 +9,14 @@
 
 public class ServiceFactory {
 
-
+private static final String SERVER_URL = "https://tribes-of-lagopus.herokuapp.com/";
```

Comment:
@georgezsiga if you have endpoints like `"/login"` you dont need the `/` at the end of the url